### ISSUE TITLE
UI: Develop Homepage - Get In Touch section

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-web#214]
## Description
As a maintainer,
I need to see BIT pages redesigned,
so that they meet the AnitaB Org guidelines.

## Mocks
Approved design idea for `Get in touch form` section (Designer Rahul Mohata)
<img width="630" alt="Screen Shot 2021-03-28 at 5 23 59 pm" src="https://user-images.githubusercontent.com/29667122/112744285-769c3500-8fea-11eb-84f3-54dbd7109cc9.png">

### Update [Required]
- [ ] Implement the design above in the section it is intended (Get In Touch section of the homepage).

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

## Additional Information

Link to [Figma file](https://www.figma.com/file/V22SXBLzMd9bqnita6hfMp/Bridge_in_tech?node-id=113%3A20)

**Comment by mtreacy002** [comment]
@Rahulm2310 , would you like to work on this issue? You can remove the OSH label if you'd like to work on it as you are the OSH mentor 😁. But this is afterall your design idea, so you have the right to claim it first 😉. If you are happy for other contributors to work on this, feel free to relabel it as `Available` and keep the 'Open Source Hack` label on.

**Comment by Rahulm2310** [comment]
@mtreacy002 I would be more than happy to give  other contributors a chance 😊

**Comment by naveen8801** [comment]
hi @mtreacy002  @Rahulm2310  , I'd like to work on it  , please assign it to me :-)

**Comment by Amulya-coder** [comment]
Yours @naveen8801 Happy coding 🎉

**Comment by Rahulm2310** [comment]
@naveen8801 Please complete the compulsary task #218  before working on anything else. Unassigning you for now. Please re-claim it once the compulsory task is done. 🙂

**Comment by battuAshita** [comment]
Hi @mtreacy002. I would like to work on this issue. Can it be assigned to me?

**Comment by Rahulm2310** [comment]
@battuAshita Thanks for showing interest. Please complete the compulsary task #218 before claiming any other issue. Thanks 😊

**Comment by battuAshita** [comment]
Yes. Will complete and get back:) Thank you

**Comment by shades-7** [comment]
can I work on this?

**Comment by Rahulm2310** [comment]
Yours @shades-7. Happy Coding 😊

**Comment by Amulya-coder** [comment]
@shades-7 you're already assigned to #203 please wait for the reviewers to review it moreover contributors will assign up to one issue atm when it's getting ready to merge then you can reclaim this issue again 😉.

**Comment by devkapilbansal** [comment]
@Amulya-coder AFAIK as long as PR is not marked as `changes requested` new issue can be assigned to the contributor.

Also, @shades-7 you can contribute by reviewing others PR too. Reviewing each other PRs will ease the work of maintainers and for sure will increase your expertise towards particular domain

**Comment by shades-7** [comment]
hello, where should i submit my prototype for this?

**Comment by Rahulm2310** [comment]
@shades-7 this issue is for implementing the UI and not designing. The designs are already approved. You need to develop this component and add it to the home page. 😊

**Comment by shades-7** [comment]
> @shades-7 this issue is for implementing the UI and not designing. The designs are already approved. You need to develop this component and add it to the home page. 😊

Do I have to make it responsive too? If yes, what's the design for mob version?

**Comment by Rahulm2310** [comment]
@shades-7 Simply try stacking the heading and the form on one another like - 
`Get in Touch`
`Subtitle`
`Form`

**Comment by Rahulm2310** [comment]
@mtreacy002 Title of the design issues sounds misleading (sounds more like design than development). Can you please update it to something more specific like `UI: Develop Homepage - Get In Touch section` ?

**Comment by shades-7** [comment]
should I make this section in the home component and can I get Figma link too?

**Comment by Rahulm2310** [comment]
@shades-7 Please create a seperate component for it and then add it to the homepage. Link to the figma file is available in the issue description.

**Comment by shades-7** [comment]
@Rahulm2310 can't find the font "meta pro" what to do?
for how many dimensions I have to write a media query?

**Comment by shades-7** [comment]
![Screenshot from 2021-04-12 17-19-21](https://user-images.githubusercontent.com/58786336/114389934-41c7da80-9bb3-11eb-8762-f6b5fd521e75.png)
This is how it looks right now with the font

**Comment by mtreacy002** [comment]
> @Rahulm2310 can't find the font "meta pro" what to do?

@shades-7 , the file to download the font `Meta Pro` was referred to in the zulip discussion [here](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv/topic/design/near/212972418).
